README.md

```diff
@@ -38,6 +38,34 @@ git clone https://github.com/$YOUR_GITHUB_USERNAME/autoscaler.git
 cd autoscaler
 ```
 
+## Building the Cluster autoscaler from master
+
+```shell
+# Replace "$YOUR_GITHUB_USERNAME" below with your github username
+git clone https://github.com/$YOUR_GITHUB_USERNAME/autoscaler.git
+# Replace "GO_Versiom" below with the version of go you want to use(eg. go1.11)
+wget https://dl.google.com/go/$GO_Version.linux-amd64.tar.gz
+# Extract the tar file
+tar -xvf go1.11.linux-amd64.tar.gz
```

Comment:
Not sure if that's redundant to have golang setups here.. I think CA doc should concentrate on build CA images?

Reply:
You use GO_Version env but here it's fixed version. This can be removed.

Reply:
@Jeffwan Updated the GoLang version  and what I feel is there might be people , sys admins you actually might not know GO , so if little generic steps are provided for setting up go and building the binary from the master to test out the latest features from the branch , it would be helpful IMO